modules/gov/handle_block.go

```diff
@@ -70,11 +70,9 @@ func updateParams(height int64, govClient govtypes.QueryClient, db *database.Db)
 	}
 
 	return db.SaveGovParams(types.NewGovParams(
-		govtypes.NewParams(
-			votingRes.GetVotingParams(),
-			tallyRes.GetTallyParams(),
-			depositRes.GetDepositParams(),
-		),
+		types.NewvotingParams(votingRes.GetVotingParams()),
+		types.NewdepositParam(depositRes.GetDepositParams()),
+		tallyRes.GetTallyParams(),
```

Comment:
Why are you not converting tally params as well? I think it's better to have consistency overall and convert them as well if needed